src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -70,6 +70,7 @@ public static partial class PlatformDetection
         public static bool IsRiscV64Process => (int)RuntimeInformation.ProcessArchitecture == 9; // Architecture.RiscV64;
         public static bool IsX64Process => RuntimeInformation.ProcessArchitecture == Architecture.X64;
         public static bool IsX86Process => RuntimeInformation.ProcessArchitecture == Architecture.X86;
+        public static bool IsLoongArch64Process => RuntimeInformation.ProcessArchitecture == Architecture.LoongArch64;
```

Comment:
`Architecture.LoongArch64` was added in net7.0, I think it should be recognizable here.

Reply:
You will need to guard it to exclude .NET Framework.

```c#
        public static bool IsLoongArch64Process =>
#if NET
			RuntimeInformation.ProcessArchitecture == Architecture.LoongArch64;
#else
			false;
#endif
```

Reply:
Thanks, we'd better keep `public static bool IsLoongArch64Process => (int)RuntimeInformation.ProcessArchitecture == 6; // Architecture.LoongArch64;` here?

Reply:
This file is already using preprocessor directive. It would be more idiomatic to use that way without the numeric values.

Reply:
The version suggested by @am11 is better. It avoids hardcoded constant. You can also change the RiscV one to match while you are on it.

Reply:
Thanks, I have modified LA64 and RV64 follow the write of @am11 .Could you please take a look?
cc @clamp03

Reply:
@LuckyXu-HF Thank you. As far as I know `Architecture.RiscV64` is not added to `net8.0`, so builds failed in CI Checks with `'Architecture' does not contain a definition for 'RiscV64'` error msg. IMO, RiscV64 should be kept as it is. @am11 @jkotas Could you share any idea?

Reply:
@clamp03 it was reverted later.